docs/001-about.md

```diff
@@ -24,5 +24,25 @@ With OpenVoiceOS, you have the option to run the platform fully offline, giving
 
 So if you're looking for a personal assistant and smart speaker that gives you the freedom and control you deserve, be sure to check out OpenVoiceOS today!
 
+## A brief history about OpenVoiceOS
+
+OpenVoiceOS (OVOS) is a collective of programmers and hardware enthusiasts who produce an open-source voice assistant. Our core team began in 2019-2020 as an offshoot of the Mycroft community, bringing a handful of third-party projects under one roof. One of those projects was an extension of Mycroft itself, which slowly became independent of its parent. Some years later, our assistant is brimming with added features, and we maintain an extensive catalog of related software.
+
+We've been positioned to do this, in part, because the team consists of well-known Mycroft contributors, several of whom work or have worked on Mycroft-derived projects for a living. Our credibility within the Mycroft community helped us attract a small but dedicated community of our own, and we've been operating at a comfortable scale ever since.
+
+In the early 2020s, Mycroft's first-party codebase began to stale, as its developers became increasingly focused on a bespoke implementation for their Mark II smart speaker. Other Mycroft implementations - some commercial, some run by open-source communities - began to struggle with Mycroft's stale codebase, and several turned to us for an actively-maintained alternative. With their employers' consent, those of us paid to work on relevant projects migrated to OVOS, which enabled those members to spend company time on our open-source projects. In this roundabout fashion, the OVOS assistant, which will receive a name of its own in the coming weeks, became an accidental fork of Mycroft. We've been operating in this manner for over two years, without giving the arrangement much thought.
+
+
+## Vision & Goals of Organization
+
+At OpenVoiceOS, we envision a world where technology empowers individuals to navigate their daily lives with privacy, security, and ease.
+
+Our vision is to create a fully community-driven, open-source, privacy-respecting voice assistant that puts users in control of their data and empowers them to interact with their technology in a way that feels natural and intuitive.
+
+We believe that by collaborating with a diverse range of contributors, we can create an accessible and inclusive platform that supports a wide range of languages and abilities.
+
+Through transparency and ethical practices, we aim to promote trust and confidence in the tech industry and to provide a platform that empowers individuals to navigate their digital lives with ease and security.
+
+We welcome and encourage both commercial and nonprofit organizations to build upon our software. We are ready to offer assistance, provided they share our commitment to privacy and user control
 
 **Disclaimer**: This post was written in collaboration with ChatGPT
```

Comment:
Does all of it was written by AI?

Reply:
> Does all of it was written by AI?

It is copied from https://www.gofundme.com/f/openvoiceos